ui/apps/platform/src/Containers/VulnMgmt/List/Nodes/VulnMgmtListNodes.js

```diff
@@ -59,7 +59,7 @@ export function getNodeTableColumns(workflowState) {
                         url={url}
                         fixableUrl={fixableUrl}
                         hideLink={pdf}
-                        imageNotes={notes}
+                        nodeNotes={notes}
```

Comment:
If we generalize this,
```suggestion
                        notes={notes}
```